src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  margin: 0px;
```

Comment:
if you write `0`, no need to write `px, rem ...`, it`s a mistake for browser.
```suggestion
  margin: 0;
```